### PR TITLE
Trivial: formatting

### PR DESCRIPTION
**Pull request by John-Colvin**

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

File: src/dmd/cond.d [hunk 1]
```diff
@@ -118,7 +118,7 @@ extern (C++) final class StaticForeach : RootObject
      */
     bool needExpansion = false;
 
-    extern (D) this(const ref Loc loc,ForeachStatement aggrfe,ForeachRangeStatement rangefe)
+    extern (D) this(const ref Loc loc, ForeachStatement aggrfe, ForeachRangeStatement rangefe)
     {
         assert(!!aggrfe ^ !!rangefe);
```
Comment (RazvanN7) [inline src/dmd/cond.d:123]: I have a headache trying to understand what this assert tests. Maybe refactor this line also while you're at it?

Comment (Geod24) [reply src/dmd/cond.d:123]: It just tests that only one is true. `!!` is a common way to get the boolean value, and XOR is XOR.
The equivalent using `&&` and `||` wouldn't be much prettier:
```D
assert((aggrfe is null) != (rangefe is null));
```
The above is the shortest expanded version I can think of.

Comment (RazvanN7) [reply src/dmd/cond.d:123]: It is larger, but it's much clearer IMHO.

**Comment by ibuclaw** [comment]
1. Druntime and phobos has a style linter pipeline. Perhaps you could add that to dmd and fix warnings flagged?

2. Commit messages should really be longer than one word. We've been using git for how long now? No need to use the old svn-isms.

File: src/dmd/cond.d (hunk 1 above)
Comment (WalterBright) [reply src/dmd/cond.d:123]: It all depends on what you're used to. Getting used to shorthand idioms is very useful in the long term.

**Pull request merged**